Tests/TuistGeneratorTests/Linter/TargetLinterTests.swift

```diff
@@ -218,7 +218,7 @@ final class TargetLinterTests: TuistUnitTestCase {
         XCTContainsLintingIssue(
             result,
             LintingIssue(
-                reason: "Target \(bundle.name) cannot contain sources. iOS bundle targets don't support source files",
+                reason: "Target \(bundle.name) cannot contain sources. bundle targets in one of these destinations doesn't support source files: iPad, iPhone, macWithiPadDesign",
```

Comment:
Nit: `bundle` ~> `Bundle`

Reply:
Is `macWithiPadDesign` a user-facing type?

Reply:
It will need to be, in Xcode it's `Mac (Designed for iPad)`.   there is a similar one for Vision in Xcode 15.

Once we expose support in `ProjectDescription` there will be a user facing type here.  Not sure what we may want to do in the migration period when the overloaded iOS deployment target is still what people use.

Reply:
thinking some, might be able to update this specific error case because it is more platform related than destination related.